monkestation/code/modules/blood_datum/components/advert_force_speak.dm

```diff
@@ -0,0 +1,134 @@
+/datum/component/advert_force_speak
```

Comment:
why is this in the blood_datum folder

Reply:
I think I got confused by the presence of `/datum/component/item_receiver` and `/datum/component/limbless_aid` in that directory
After a bit of of looking around I've spotted a few more files which shouldn't be in blood_datum which I'd be happy to cleanly refactor as part of this PR

Reply:
feel free